packages/fields/src/fields/title/title-view.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * WordPress dependencies
+ */
+import { __experimentalHStack as HStack } from '@wordpress/components';
+import { decodeEntities } from '@wordpress/html-entities';
+import { __ } from '@wordpress/i18n';
+import { useSelect } from '@wordpress/data';
+import { store as coreStore } from '@wordpress/core-data';
+import type { Settings } from '@wordpress/core-data';
+
+/**
+ * Internal dependencies
+ */
+import type { BasePost } from '../../types';
+import { getItemTitle } from '../../actions/utils';
+
+const TitleView = ( { item }: { item: BasePost } ) => {
+	const { frontPageId, postsPageId } = useSelect( ( select ) => {
+		const { getEntityRecord } = select( coreStore );
+		const siteSettings: Settings | undefined = getEntityRecord(
+			'root',
+			'site',
+			'',
+			undefined
+		);
```

Comment:
This was `const siteSettings = getEntityRecord( 'root', 'site' );` in the JavaScript version.

Reply:
I think for third-argument it's actually clearer because `getEntityRecord` requires an id in general. But I believe we can make the fourth argument optional `=something` in the type definition to avoid having to pass it.

Reply:
I've pushed https://github.com/WordPress/gutenberg/pull/66940/commits/cda60872330dd6f29ee2e20bae30a5e7d877a607

Reply:
> I've pushed https://github.com/WordPress/gutenberg/commit/cda60872330dd6f29ee2e20bae30a5e7d877a607

Reverted this at fec23a7ce50d095b641f5b49e3d33c7fe952a1d9 TypeScript doesn't complain about the missing last argument.